### PR TITLE
fix(dashboard-detail): update origin state in edit-name

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

[QA - 319 - 2](https://pyengine.atlassian.net/browse/QA-319)

Update origin state too, when edit dashboard name in detail page.

### CODE REVIEW

File: src/services/dashboards/dashboard-detail/DashboardDetailPage.vue [hunk 1]
```diff
@@ -181,6 +181,7 @@ const handleVisibleNameEditModal = () => {
 };
 const handleNameUpdate = (name: string) => {
     dashboardDetailState.name = name;
+    dashboardDetailOriginState.dashboardInfo.name = name;
```
Comment (yuda110) [inline src/services/dashboards/dashboard-detail/DashboardDetailPage.vue:184]: suggestion: Oh, it's not good idea to change origin state. 
1) get dashboard api when update name
2) or, use `dashboardDetailState.name` in name space.

Comment (WANZARGEN) [inline src/services/dashboards/dashboard-detail/DashboardDetailPage.vue:184]: question: I thought origin state must be updated only by api.. Is this the only way?

**Review by WANZARGEN — COMMENTED** [review]
I left a question!

**Comment by piggggggggy** [comment]
Not to edit directly origin state, edit `revertDashboardData` used in detail page `onUnmounted`.

**Comment by piggggggggy** [comment]
**Redefine**

Rule
- Do not change directly origin state
- Use actions to change origin state 

- Create and apply 'setOriginDashboardName`

**Review by yuda110 — APPROVED** [review]
LGTM!!

File: src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ const DASHBOARD_DEFAULT = Object.freeze<{ settings: DashboardSettings }>({
 });
 
 export const useDashboardDetailInfoStore = defineStore('dashboard-detail-info', () => {
+    // CAUTION: don't directly access and modify originState outside this store.
```
Comment (WANZARGEN) [inline src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts:79]: praise: nice comment!

**Review by WANZARGEN — APPROVED** [review]
GOOD

**Pull request merged**